src/charm.py

```diff
@@ -157,6 +157,17 @@ def _owasp_modsecurity_crs(self):
         """Return a boolean indicating whether OWASP ModSecurity CRS is enabled."""
         return self._get_config_or_relation_data("owasp-modsecurity-crs", False)
 
+    @property
+    def _owasp_modsecurity_custom_rules(self):
+        """Return the owasp-modsecurity-custom-rules value from config or relation"""
+        custom_rules = self._get_config_or_relation_data("owasp-modsecurity-custom-rules", "")
+        # Since expected value is something like the example below,
+        # we replace the escaped new line \\n for \n
+        # SecAction "id:900130,phase:1,nolog,pass,t:none,setvar:tx.crs_exclusions_wordpress=1"\n
+        custom_rules.replace("\\n", "\n")
```

Comment:
I could have missed it, I did not spot any tests that checked for this?

Reply:
Good catch. I'll do this one on Monday.